index.js

```diff
@@ -0,0 +1,22 @@
+function openNav() {
+  document.getElementById("mySidepanel").style.width = "200px";
+}
+
+function closeNav() {
+  document.getElementById("mySidepanel").style.width = "0";
+}
+
+let task = [];
+
+console.log(task);
```

Comment:
This is unnecessary as we know for sure that task is an empty array.

Reply:
That's right.I'll fix it.